javascript/README.md

```diff
@@ -262,29 +262,29 @@ Blank line between case statements in switch
 // good
 
 switch(foo) {
```

Comment:
Should we call out that the rule of keeping `case` statements at the same indentation level of `switch`?

Reply:
sure, done.